### DISCUSSION TITLE
Not able to run taiko using firefox on windows .Please kindly suggest

### DISCUSSION THREAD

**Discussion by karishmayadava23** [getgauge/taiko#2694] [Development]
Hi ,

Tried running taiko on firefox/windows11. But unable to run followed below steps -
1. Installed firefox nightly from https://www.mozilla.org/en-US/firefox/all/#product-desktop-nightly
2.  on vscode terminal wrote below code to run the taiko in repl mode
TAIKO_BROWSER_PATH= C:\\Program Files\\Firefox Nightly\\firefox.exe  npx taiko 
but getting below error as it is not recognizing the executable path of firefox executable file.
Please suggest as providing the correct path
![image](https://user-images.githubusercontent.com/59534066/230838949-c8db117b-6793-40c5-8f26-29346fd19776.png)

**Comment by zabil** [comment]
If you are using powershell, please use the powershell syntax for setting environment variables
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.3

**Reply by karishmayadava23** [reply]
Thanks it worked for me
